packages/babel-plugin-transform-es2015-arrow-functions/test/fixtures/arrow-functions/super-in-class-method/expected.js

```diff
@@ -0,0 +1,26 @@
+class SuperClass {
+  doStuff() {
+    return "superclass";
+  }
+}
+
+class MyClass extends SuperClass {
+  superDotAccess() {
+    var _super = super;
```

Comment:
`var _super_doStuff = super.doStuff;` probably isn't a good idea because the parent could have `get doStuff(){ return function(){}; }` instead of a normal method.

Adding a new method is an interesting case I definitely hadn't considered. That would work for `super.foo` but would still fail for `super()`. Still on the fence though.

I'm curious, what led to this discovery? Were you specifically running arrow without class? I'd probably expect anything that implements classes to also have implemented arrows.